pkg/tests/upgrade_tests/upgrade_test.go

```diff
@@ -152,7 +188,7 @@ func getUpgradedDbInfo(t *testing.T, noData bool, useEarliest bool, extensionSta
 		},
 		/* postUpgrade */
 		func(dbContainer testcontainers.Container, dbTmpDir string) {
-			connectURL := testhelpers.PgConnectURL(*testDatabase, testhelpers.NoSuperuser)
+			connectURL := testhelpers.PgConnectURL(*testDatabase, testhelpers.Superuser)
```

Comment:
Hmm I think this should only apply to the snapshot part not to the data ingestion part.

Reply:
addressed in 5ce1a3cb